app/src/components/account/EmailAuthentication.vue

```diff
@@ -1,42 +1,46 @@
 <template>
-  <TmPage data-title="email-authentication">
-    <div class="card">
-      <h1 v-if="account.userSignedIn" class="authentication-title">
-        You are now signed in! &#128640;
-      </h1>
-      <h1 v-else-if="account.signInError" class="authentication-title">
-        {{ account.signInError.message }} 🙀
-      </h1>
-      <h1 v-else class="authentication-title">
-        Good bye, see you soon! &#128075;
-      </h1>
+  <SessionFrame hide-back>
+    <h2 class="session-title">
+      Magic link success!
+    </h2>
+    <div class="session-main">
+      <p class="session-subtitle">
+        You're now signed in to Lunie with your email address. Head over to the
+        notifications page to see some of your recent events.
+      </p>
+      <TmBtn @click.native="goToNotifications" value="Let's Go!" centered />
     </div>
-  </TmPage>
+
+    <template v-if="account.signInError || account.signInEmailError">
+      <h2 class="session-title">Magic link error! 🙀</h2>
+      <div class="session-main">
+        <p class="session-subtitle">{{ account.signInError.message }}</p>
+      </div>
+    </template>
+  </SessionFrame>
 </template>
 
 <script>
-import TmPage from "common/TmPage"
+import SessionFrame from "common/SessionFrame"
+import TmBtn from "common/TmBtn"
 import { mapState } from "vuex"
 export default {
   name: `email-authentication`,
   components: {
-    TmPage,
+    SessionFrame,
+    TmBtn,
   },
   computed: {
     ...mapState([`account`]),
   },
   mounted() {
     this.$store.dispatch(`signInUser`, window.location.href)
   },
+  methods: {
+    goToNotifications() {
+      this.$router.push({ name: "notifications" }).catch(err => {
```

Comment:
apparently we should be `.catch`-ing all `router.push`'s now https://github.com/vuejs/vue-router/issues/2873

Reply:
Do we use next(false)?

Reply:
well, that explains all the uncaught Promises appearing on the console then

Reply:
I guess this is the issue that we see in Lunie: https://github.com/vuejs/vue-router/issues/2873#issuecomment-549409726